nemo/core/classes/modelPT.py

```diff
@@ -1754,3 +1742,20 @@ def cuda(self, device=None):
         elif isinstance(device, int):
             device = torch.device("cuda", index=device)
         return super().cuda(device=device)
+
+    def _get_optim_config_copy(self, optim_config: Optional[Union[DictConfig, Dict]]) -> Optional[DictConfig]:
```

Comment:
Why not just "_optim_copy_config"

Reply:
Happy to rename it but did you really mean `_optim_copy_config()`, or would it be `_optim_config_copy()` / `_copy_optim_config()`?

Reply:
_optim_config_copy is better

Reply:
Sounds good, done in 8e556a9dc